Resources/Views/ProfileImageView/README.md

```diff
@@ -0,0 +1,60 @@
+# ProfileImageView
+
+A `View` subclass intended to replace `UIImageView` and all the configuration that goes with it for user's profile image.
+
+### Example: Implementation in SwiftUI
+```
+struct ContentView: View {
+  var profileImageView = ProfileImageView()
+  
+  // Optional - For configuration purposes
+  init() {
+    profileImageView.properties.placeHolder = Image(systemName: "person.circle.fill")
+    profileImageView.imageTapped = { print("Image tapped") }
```

Comment:
I few more comments about the behavior:
- `profileView.properties.backgroundType = .plain(.red)` doesn't properly set the background. Background should be only inside the circle
<img width="171" alt="Screenshot 2022-03-21 at 08 30 17" src="https://user-images.githubusercontent.com/677166/159219750-7ac418bc-757e-4bd7-a8be-da6246111b7e.png">

- border is set incorrectly. It should be set on the circle
<img width="164" alt="Screenshot 2022-03-21 at 08 32 20" src="https://user-images.githubusercontent.com/677166/159219946-85d7f709-98d1-42a9-ae7c-880a919af8e4.png">

- if I do not set `placeHolder`, there is some a default picture in there
<img width="182" alt="Screenshot 2022-03-21 at 08 34 17" src="https://user-images.githubusercontent.com/677166/159220137-d058bae8-2f54-4eaa-bc22-c08b5a77b9d2.png">

- cornerRadius should be an enum defining perfect "circle" or custom value, so if i want a circle i don't want to handle the radius all the time when the view resizes